db/util_testing.go

```diff
@@ -787,3 +787,13 @@ func RawDocWithInlineSyncData(_ testing.TB) string {
 }
 `
 }
+
+// DisableSequenceWaitOnDbStart disables the release sequence wait on db start.  Appropriate for tests
+// that make changes to database config after first startup, and don't assert/require on sequence correctness
+func DisableSequenceWaitOnDbRestart(tb testing.TB) {
```

Comment:
This is just to speed up this test, right?

I wonder if a better fix is to disable this _everywhere_ via `NewRestTester` unless it is declared it is needed. This would speed up tests by a bit. Do you think this is worth filing a separate ticket to address for other tests that are affected?

Reply:
In general this doesn't slow down most tests because the wait only happens if initialSequence > 0 (i.e. only tests where the database gets restarted after some sequence allocation work has been done, which is relatively rare), so I didn't want to immediately add it to the default RestTester.  

However, I agree it should get evaluated further - filed [CBG-4124](https://issues.couchbase.com/browse/CBG-4124)